### ISSUE TITLE
Remove "Additional Information/Identifiers" for SB ID's

### ISSUE THREAD

**Issue by zephyrr** [adiwg/mdEditor#137]
### Issue Description:

If there is a ScienceBase ID (as assigned when publishing an item without ID to ScienceBase and thus creating it), then the ScienceBase page show the ScienceBase IdD  under "Additional Information/Identifiers" with type "Identifier imported from ScienceBase during publication"

This seems redundant and clutters the page with useless information.  If there is a ScienceBase identifier in the record, it can be removed during translation, keeping only non-SB identifiers in this field.

### Steps to reproduce (Bug reports only):

  1. _____
  2. _____
  3. _____

### Observed Results (Bug reports only):

  *What happened?  This could be a description, log output, etc.*

### Expected Results (Bug reports only):

  *What did you expect to happen?*

**Comment by jllong-usgs** [comment]
These identifiers are required for other systems that access these records, such as DEPTH.

**Comment by zephyrr** [comment]
Let me clarify.  View https://www.sciencebase.gov/catalog/item/59fab476e4b0531197affd6b and scroll down to section "Additional Information" subsection "Identifiers".  In imported item (but no other items in ScienceBase), this contains an additional redundant reference to the ScienceBase identifier of this same item.  

Obviously this redundant extra copy of the SB ID is not needed by DEPTH, as nothing else in ScienceBase has it.  So I'm suggesting that when mdTranslator is translating mdJSON, it could omit creating this additional ScienceBase id.

This issue doesn't break anything so it's not first priority, but when mdTranslator is being fixed (see other issues), it would be a nice cleanup to also remove this unneeded and distracting redundancy.

**Comment by zephyrr** [comment]
Start of an sbJSON file follows.  The reference I'm questioning is the FOURTH reference to itself, which gets displayed in the "Additional Information" section.

```
{
   "link": {
        "rel": "self",
        "url": "https://www.sciencebase.gov/catalog/item/59fab476e4b0531197affd6b"
    },
    "relatedItems": {
         "link": {
           "url": "https://www.sciencebase.gov/catalog/itemLinks?itemId=59fab476e4b0531197affd6b",
           "rel": "related"
         }
       },
       "id": "59fab476e4b0531197affd6b",
       **"identifiers": [
            {
               "type": "Identifier imported from ScienceBase during publication",
               "scheme": "gov.sciencebase.catalog",
               "key": "59fab476e4b0531197affd6b"
             }**
         ],
        ...
```

**Comment by jlblcc** [comment]
@stansmith907 Thoughts?
@zephyrr Might be best to move this issue to [adiwg/mdTranslator](/adiwg/mdTranslator/issues/new)

**Comment by stansmith907** [comment]
Removed identifier where identifiers.key matches id.

**Comment by jlblcc** [comment]
Fixed in adiwg/mdTranslator#162
